mipengine/node/tasks/common.py

```diff
@@ -35,15 +32,33 @@ def get_node_info(request_id: str):
         port=node_config.rabbitmq.port,
         db_ip=node_config.monetdb.ip,
         db_port=node_config.monetdb.port,
-        datasets_per_data_model=datasets_per_data_model,
     )
 
     return node_info.json()
 
 
 @shared_task
 @initialise_logger
-def get_data_model_cdes(request_id: str, data_model: str) -> Dict[str, str]:
+def get_node_datasets_per_data_model(request_id: str) -> Dict[str, Dict[str, str]]:
+    """
+    Parameters
+    ----------
+    request_id : str
+        The identifier for the logging
+    Returns
+    ------
+    Dict[str, Dict[str, str]]
+        A dictionary with key data model and value a list of pairs (dataset code and dataset label)
+    """
+    return {
+        data_model: get_data_model_datasets(data_model)
```

Comment:
I am not really sure what `get_data_model_datasets` returns. It's not only datasets right, it's the labels as well?